www/docs/style/sass/pages/_business.scss

```diff
@@ -0,0 +1,378 @@
+    line-height: 1.2em;
+}
+
+.business-section__header__description {
+    p {
+        margin: 0.5em 0 0 0;
+        max-width: 36em;
+        color: mix($primary-color, #fff, 25%);
+    }
+}
+
+.business-section__solo {
+    @include grid-column(12, $collapse: true);
+    padding: 1.5em;
+
+    @media (min-width: $medium-screen) {
+        padding: 2.5em 3em;
+    }
+}
+
+.business-section__primary {
+    @include grid-column(12, $collapse: true);
+    padding: 1.5em;
+    border-bottom: 1px solid $colour_off_white;
+
+    @media (min-width: $medium-screen) {
+        padding: 2.5em 3em;
+    }
+
+    @media (min-width: $large-screen) {
+        @include grid-column(8, $collapse: true);
+        padding: 2.5em 3em;
+        border-bottom: none;
+        border-right: 1px solid $colour_off_white;
+    }
+}
+
+.business-section__secondary {
+    @include grid-column(12, $collapse: true);
+    padding: 1.5em;
+
+    @media (min-width: $medium-screen) {
+        padding: 2.5em 3em;
+    }
+
+    @media (min-width: $large-screen) {
+        @include grid-column(4, $collapse: true);
+        padding: 2.5em 3em;
+    }
+
+    .rss-feed {
+        margin-top: 2em;
+        font-size: 0.8em;
+    }
+}
+
+.business-list {
+    @include unstyled-list();
+    margin: 0;
+
+    li {
+        & + li {
+            margin-top: 2em;
+        }
+
+        & > :last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    ul {
+        @include unstyled-list();
+        margin: 0;
+
+        li {
+            margin-top: 1em;
+            border-left: 0.2em solid $body-bg;
+            padding-left: 1em;
+        }
+
+        li + li {
+            margin-top: 1.5em;
+        }
+    }
+}
+
+.business-list__title {
+    display: block;
+    margin: 0 0 0.3em 0;
+
+    h3 {
+        color: inherit;
+        margin: 0;
+        line-height: 1.2em;
+    }
+}
+
+.business-list__meta {
+    font-size: emCalc(14);
+    color: $colour_grey;
+    margin-bottom: 0.5em;
+}
+
+.business-list__excerpt {
+    font-size: 0.9em;
+}
+
+.calendar__controls {
+    @include grid-row();
+    @extend .upcoming__controls;
+    margin-bottom: 1em;
+}
+
+.calendar__controls__previous,
+.calendar__controls__next {
+    @include grid-column(2);
+}
+
+.calendar__controls__previous {
+    @extend .controls__prev;
+}
+
+.calendar__controls__next {
+    @extend .controls__next;
+}
+
+.calendar__controls__current {
+    @include grid-column(8);
+    @extend .controls__current;
+}
+
+.calendar {
+    table {
+        margin-bottom: 0;
+        border: none;
+        background: transparent;
+        text-align: center;
+        width: 100%;
+    }
+
+    thead {
+        background: transparent;
+    }
+
+    th {
+        text-align: inherit;
+    }
+
+    td {
+        padding: 0;
+        border-bottom: 1px solid $colour_off_white;
+
+        & > * {
+            display: block;
+            padding: 0.5em 0.55556em;
+        }
+
+        &.no {
+            color: $light-text;
```

Comment:
Don't know if this is my monitor, but I can't really tell the difference at all between `.no` days and other days in the calendar unless I squint.